Ports/chocolate-doom/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='chocolate-doom'
+version='3.0.1'
+useconfigure='true'
+squashed_version=$(echo "${version}" | tr -d '.')
+files="https://www.chocolate-doom.org/downloads/${version}/chocolate-doom-${version}.tar.gz chocolate-doom-${version}.tar.gz d435d6177423491d60be706da9f07d3ab4fabf3e077ec2a3fc216e394fcfc8c7"
+auth_type='sha256'
+workdir="chocolate-doom-${version}"
+depends=("libpng" "libsamplerate" "SDL2" "SDL2_mixer" "SDL2_net")
```

Comment:
Please put these dependencies on their own lines and use single quotes instead of double quotes.
```suggestion
depends=(
    'libpng'
    'libsamplerate'
    'SDL2'
    'SDL2_mixer'
    'SDL2_net'
)
```